### ISSUE TITLE
ServiceManager is not present on jobs run without gearman

### ISSUE THREAD

**Issue by tomaszdurka** [cargomedia/cm#2404]
introduce a service on which we queue the jobs?
Similar like we do it for the delayed jobs:
```php
$delayedJobQueue = CM_Service_Manager::getInstance()->getDelayedJobQueue();
$delayedJobQueue->addJob(new CM_User_OfflineJob(), ['user' => $user], CM_Model_User::OFFLINE_DELAY);
```

cc @alexispeter

**Comment by alexispeter** [comment]
Would be nice.

**Comment by tomaszdurka** [comment]
This might be related: https://github.com/cargomedia/cm/pull/1659

**Comment by alexispeter** [comment]
I think it would make sense to only provide one service for all usages of jobs. Something like a `jobManager` as illustrated in the second example, providing methods for running, queueing, and delayed queueing.

**Comment by alexispeter** [comment]
How about, in the meantime, we just add the serviceManager directly in the code where they're instantiated and run?

**Comment by alexispeter** [comment]
Only to those jobs that need it of course.

**Comment by tomaszdurka** [comment]
We can do that. Yeah, the thing is the ServiceManager should be on all the jobs with current implementation. We don't set it as it's not needed, but there is getter and we don't really know if it's gonna be used (as seen in the described case).

**Comment by tomaszdurka** [comment]
This should be solved by #1659.
